packages/react-native/bugsnag-react-native-xcode.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 DEST="$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH"
 
-BUNDLE_FILE="$DEST/main.jsbundle"
+BUNDLE_FILE="$CONFIGURATION_BUILD_DIR/main.jsbundle"
```

Comment:
So this path works for non-Hermes React Native too?

Reply:
Yeah, I checked it.

For both Hermes and non-Hermes, the RN Xcode build script sets its own `BUNDLE_FILE` variable [here](https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/scripts/react-native-xcode.sh#L118).

For non-Hermes, it then [copies the `BUNDLE_FILE` into `$DEST/main.jsbundle`](https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/scripts/react-native-xcode.sh#L163-L164).

For Hermes, it [compiles the JS bundle into an .hbc (Hermes byte-code) bundle directly in the `$DEST/main.jsbundle` location](https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/scripts/react-native-xcode.sh#L175).

Then it [bundles the JS](https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/scripts/react-native-xcode.sh#L157) into that location.

Hope that makes sense!